src/test/java/FizzBuzzTest.java

```diff
@@ -41,24 +42,24 @@ private void doFizzBuzz(int n, String s) throws IOException {
 		System.out.flush();
 		assertEquals(s, baos.toString());
 	}
+	
+	private String generateFizzBuzzTestStringsUntil20 (int length) {
+		if (length < 1 || length > 20) {
+			throw new IllegalArgumentException("generateFizzBuzzTestStringsUntil20 must be called with a length from 1-20");
+		}
+		String[] outputs = {"1", "2", "Fizz", "4", "Buzz", "Fizz", "7", "8", "Fizz", "Buzz", "11", "Fizz", "13", "14", "FizzBuzz", "16", "17", "Fizz", "19", "Buzz"};
+		String output = "";
+		for(String s: Arrays.asList(outputs).subList(0, length)) {
+			output = output + String.format(s + "%n");
```

Comment:
I fear we are creating a tight coupling to the decisions made by `String.format` regarding line breaks here. Ideally we should leverage [existing work on separation of line break formatting into strategies](https://github.com/EnterpriseQualityCoding/FizzBuzzEnterpriseEdition/blob/master/src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/stringreturners/NewLineStringReturner.java) or create a new strategy if the concerns of this line break are found to be separate in nature from those of other line breaks.